modules/apps/style-book/style-book-web/src/main/resources/META-INF/resources/js/StyleBookManagementToolbarPropsTransformer.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {openSimpleInputModal} from 'frontend-js-web';
+
+export default function propsTransformer({
+	additionalProps: {copyStyleBookEntryURL, exportStyleBookEntriesURL},
+	portletNamespace,
+	...otherProps
+}) {
+	const copySelectedStyleBookEntries = () => {
+		const form = document.getElementById(`${portletNamespace}fm`);
+
+		const styleBookEntryIds = document.getElementById(
+			`${portletNamespace}styleBookEntryIds`
+		);
+
+		if (!form && !styleBookEntryIds) {
+			return;
+		}
+
+		styleBookEntryIds.setAttribute(
+			'value',
+			Liferay.Util.listCheckedExcept(form, `${portletNamespace}allRowIds`)
+		);
+
+		const styleBookEntryFm = document.getElementById(
+			`${portletNamespace}styleBookEntryFm`
+		);
+
+		if (!styleBookEntryFm) {
+			return;
+		}
+
+		submitForm(styleBookEntryFm, copyStyleBookEntryURL);
+	};
+
+	const deleteSelectedStyleBookEntries = () => {
+		if (
+			confirm(
+				Liferay.Language.get('are-you-sure-you-want-to-delete-this')
+			)
+		) {
+			const form = document.getElementById(`${portletNamespace}fm`);
+
+			if (!form) {
+				return;
+			}
+
+			submitForm(form);
```

Comment:
That being said @markocikos about the `if` check, why are we doing:
```
			const form = document.getElementById(`${portletNamespace}fm`);

			if (!form) {
				return;
			}

			submitForm(form);
```

Shouldn't we also do the same for :
```
const form = document.getElementById(`${portletNamespace}fm`);

if (form) {
    submitForm(form);
}
```

Reply:
Yep :+1:

Reply:
Ok I think when everything is done, I will go through all the components and make sure these SF nit changes are made for the `if` check.  It seems we are using some combination of them throughout depending on who is making the changes.